probe-rs/src/probe/espusbjtag/protocol.rs

```diff
@@ -10,8 +10,8 @@ use crate::{
 const JTAG_PROTOCOL_CAPABILITIES_VERSION: u8 = 1;
 const JTAG_PROTOCOL_CAPABILITIES_SPEED_APB_TYPE: u8 = 1;
 const MAX_COMMAND_REPETITIONS: usize = 1024;
-const OUT_BUFFER_SIZE: usize = OUT_EP_BUFFER_SIZE * 32;
-const OUT_EP_BUFFER_SIZE: usize = 128;
+const OUT_BUFFER_SIZE: usize = OUT_EP_BUFFER_SIZE * 2; // each command is 4 bits, i.e. 2 commands per byte
```

Comment:
I prefer comments above the code instead of the same line for readability (not all code is like this currently but I actively fix it when I encounter it).